src/main/scala/io/joern/typestubs/ruby/BuiltinPackageDownloader.scala

```diff
@@ -94,6 +100,7 @@ class BuiltinPackageDownloader(outputDir: String, format: OutputFormat.Value = O
   ): Iterator[() => (String, List[RubyType])] = {
     logger.info("[Ruby]: Generating Ruby Types for builtin functions")
     pathsMap
+      .slice(0, 5)
```

Comment:
What is this for?

Reply:
Good catch, removed. Was using it to test out changes for constructor names so that I don't have to wait for the whole site to be scraped.